application/src/main/java/org/opentripplanner/apis/gtfs/configure/SchemaModule.java

```diff
@@ -0,0 +1,27 @@
+package org.opentripplanner.apis.gtfs.configure;
+
+import dagger.Module;
+import dagger.Provides;
+import graphql.schema.GraphQLSchema;
+import jakarta.inject.Singleton;
+import org.opentripplanner.apis.gtfs.SchemaFactory;
+import org.opentripplanner.framework.application.OTPFeature;
+import org.opentripplanner.routing.api.request.RouteRequest;
+
+/**
+ * The schema is used during application serve phase, not loading, and it depends on the default
+ * route request, which is injected from the
+ * {@link org.opentripplanner.standalone.config.RouterConfig}. The {@link GraphQLSchema} is only
+ * constructed if the API feature flag is on.
+ */
+@Module
+public class SchemaModule {
+
+  @Provides
+  @Singleton
+  public GraphQLSchema provideSchema(RouteRequest defaultRouteRequest) {
```

Comment:
@Nullable - I think dagger throw a NPE if this is called with the GtfsGraphQlApi off